### DISCUSSION TITLE
Appended chapter number cannot be changed

### DISCUSSION THREAD

**Discussion by Malevola** [audiamus/AaxAudioConverter#70] [Q&A] [answered]
When I import my MP3 files into my music player (in this case MusicBee), the playing track shows "::: Chapter #" at the end of the file name. I'm not certain where this information is obtained, since there doesn't seem to be a tag for it, but this information appears for the playing track even if I import a single file without chapter splits. Oddly, It will show the last few chapter numbers throughout playback (e.g., "Book Name ::: Chapter 102" will appear at the start of a book with 105 chapters even if I imported a single file). 

Is there a setting I'm missing somewhere to eliminate what is basically pointless information?

**Comment by audiamus** [comment, marked as answer]
This will come from the embedded chapter meta data. 

Few players support chapters for MP3, MusicBee appears to be one of them. However, the chapter meta format is defined in ID3v2, the meta data specification for MP3. AAX Audio Converter transfers the chapter data either from the Audible AAX file or from the associated content meta JSON file with the help of https://github.com/Zeugma440/atldotnet.

The audiobook chapter meta data does not necessarily match the actual chapter structure in the book, it's down to the Audible book editor to set this up correctly.   

Another player that can read MP3 chapters is MPC-HC (sadly no longer under active development, latest releases in maintenance mode here: https://github.com/clsid2/mpc-hc/releases). Play your MP3 audiobook with MPC-HC to compare with MusicBee. MPC-HC will show the current chapter info in the bottom pane.

**Comment by Malevola** [comment]
Thanks for your response. I just re-encoded the files within MusicBee and the tag fell off.
